js/home.js

```diff
@@ -0,0 +1,39 @@
+const homePageData = new XMLHttpRequest();
+homePageData.open("GET", "./api/home.json");
+homePageData.send();
+homePageData.addEventListener("load", loadHomeData);
+
+function loadHomeData() {
+    const homeData = JSON.parse(homePageData.response);
+    const content = document.getElementById('page-content');
+    content.innerHTML = homeData['text-content'];
+    const headingRow = document.getElementById('table-heading');
+    for(let i=0; i<homeData['table-headings'].length; i++) {
```

Comment:
Assign the homeData['table-headings'].length to a variable and use that variable in all for loops so that the .length check is not called for all iterations